### ISSUE TITLE
Tracking issue: components and features for SDG article page

### ISSUE THREAD

**Issue by mlbrgl** [owid/owid-grapher#1721]
## Core problem

The SDG tracker has been built on different stack than ourworldindata.org, which makes it hard for us to update, keep current and improve.

## Proposed solution

We are migrating the content from sdg-tracker.org over to ourworldindata.org.
Mallika has migrated the existing content to the new CMS (see #1703). 

We now need to fill in the blanks in terms of components and features to align the rendered page with the new design

<img width="1440" alt="Screenshot 2022-10-25 at 15 10 46" src="https://user-images.githubusercontent.com/13406362/197781956-5fd2dfa1-a4ae-471f-8284-fc0bc80edbd9.png">

### Essential

- [x] #1722
  - [x] #1782
- [x] #1758 
- [x] #1733
  - [x] responsive
- [x]  #1762
- [x]  #1760
- [x]  #1761
- [x] #1779
- [x] side-by-side charts
- [ ] #1809 

### Bugfix

- [x] #1784
- [ ] #1814 
- [x] The “index” title does not look aligned to the the targets underneath
- [x]   The typography inside the index component is off for a few reasons:
  - [x]   target titles and targets themselves are more spaced out than different targets
  - [x]   target font-size should be 16px
  - [x]   the text seems like it’s bold but should be semi-bold
- [x]   Overline text “TARGET XYZ” should be #6E87A2, font-weight Black
- [x]   The body text under the H1s for example should be center aligned
- [x]   The space between the H1 and body text should be 24px not 34
- [x]   For inline links, can we use the same style as we usually have? (just underline?
- [x] #1796
- [x]   I think Matthieu said this wasn’t possible but I’ll just put it here in case: Is it possible to stretch the lines above the titles (below the blocks), from side-to-side?
    - _(Matthieu) for context, this was before the typing refactor. Now that headings are not wrapped into divs, this should be more approachable._ 
- [x]   Can we make the space between the last block and the footer 48px? it’s 30 now
- [x]   The links in the “reuse this work freely” should be the same color as the rest of the text and for some reason the underline under “CC BY” is burgundy
- [x]   I think you already know this, but the chart story component has some alignment issues: The arrows on left and right should be aligned to the sides f the chart and the spacing between the overline and the title is huge.
- [x] Heading size on mobile ([slack](https://owid.slack.com/archives/C040429G2SD/p1670496350478069))
- [ ]   In the design I had the text in 5 columns and the charts in 7. Here it looks like the text is in 6 and the chart isn’t really fitting ito the grid.
  - _discussed with Mallika on Dec 9: switching to `sticky-right` blocks_
- [ ]   The Additional charts still needs to be styled I’m guessing?
  - _(Matthieu) for context, this is part of the "nice to have" scope (see below). I might address it before launch but this is a non blocker_
- [ ]   The links in the summary in t[his page](https://ptolemy-owid.netlify.app/new-grid-format-ensure-healthy-lives-and-promote-well-being-for-all-at-all-ages) don’t look great but also the summary block itself could use a little styling. If I send you something, can we do it or is it too late? I will work on something anyway and worst case we can implement it in the future ! 🙂 But it would be nice for now at least to have links styled nicely in the summary block if that’s ok.
  - _pending discussion between @ikesau and @danyx23 on Dec 12_ 

### Nice to have

- [ ] #1802
- [x] #1800
- [ ] `risk overlap`  back to top
- [ ] `risk overlap + needs derisking` static images
- [ ] `risk overlap` expandable paragraph
- [x] moved to essential scope ⬆️ ~~`needs derisking` expandable and or dynamic table of content "index"~~
- [ ] `needs derisking` previous / next target page navigator
- [x] `risk overlap` footer?
- [ ] #1822
- [ ] #1823

## Context

This migration is done in parallel with bringing all articles from Wordpress into the new CMS.

Figma: https://www.figma.com/file/9bQW46okOjxeJtismQQdPu/%5BDesktop%5D-Screens-1.0?node-id=2052%3A18791

## Risks
- increased collaboration and conflict resolution costs due to 2 other projects evolving in the same technical space over the course  of the cycle
- shortened cycle due to off-site preparation
- first time the new CMS will be used in production for live articles: this might uncover unexpected issues and slow progress on the core work of this issue
- static image support involves architectural decisions
- depends on non-SDG specific features that might not have been taken into account when estimating (static image, publication context, breadcrumb)
- depends on styles refactor

**Comment by mlbrgl** [comment]
Moved *expandable and or dynamic table of content "index"* from "nice to have" to "essential" scope (see #1733) after evaluation of dev effort vs authoring effort.

**Comment by mlbrgl** [comment]
Moved "Footer" to "nice to have" scope

**Comment by mlbrgl** [comment]
Added bugfix items following Marwa's review

**Comment by danyx23** [comment]
@mlbrgl Should we create a new small follow-up issue for this one with just the remaining nice-to-have? Or do you prefer to keep this one around?

**Comment by mlbrgl** [comment]
@danyx23 Given the essential scope has been fully tackled, I would just close this issue for now. 

We'll need to reevaluate the "nice to have" scope to make sure it is current before creating a new issue.
